src/components/Nav.jsx

```diff
@@ -7,11 +7,8 @@ import {
   StyledOcticon,
 } from "@primer/react";
 import { TableIcon } from "@primer/octicons-react";
-import useAuthState from "../hooks/use-auth-state";
-
-export default function Nav() {
-  const { authState, logout } = useAuthState();
 
+export default function Nav({ avatarUrl, onLogout }) {
```

Comment:
@gr2m just FYI: I ended up using `props` for this component (so it's the responsibility of the parent to pass values associated to the expected `authState`).

Because it's the only occurrence of `<Nav>` component I think it makes sense using props for now. The moment we see the need to use this component in more places and passing the `props` down becomes tedious, we can migrate it to use React's [`Context`](https://react.dev/learn/passing-data-deeply-with-context).